_dev/.stylelintrc

```diff
@@ -4,7 +4,8 @@
     "stylelint-config-recommended-scss"
   ],
   "ignoreFiles": [
-    "../packages/**/tailwind.out.scss"
+    "../**/tailwind.out.scss",
+    "../libs/accounts/email-renderer/src/**/*.{scss,css}"
```

Comment:
I kept getting errors on git commit... Not sure why... Maybe this needs a followup.